core/controllers/editor_test.py

```diff
@@ -1580,6 +1580,64 @@ def test_get_with_invalid_version_raises_error(self):
             params={'v': 'invalid_version'}, expected_status_int=404)
         self.logout()
 
+    def test_put_with_long_commit_message_raises_error(self):
```

Comment:
This test seems way more complex than necessary to test what you are testing. Please reduce it to the simplest possible test.

Reply:
Done